cmd/context/create.go

```diff
@@ -128,7 +128,8 @@ func (c *ContextCommand) UseContext(ctx context.Context, ctxOptions *ContextOpti
 		} else {
 			if !isValidCluster(ctxOptions.Context) {
 				return oktetoErrors.UserError{E: fmt.Errorf("invalid okteto context '%s'", ctxOptions.Context),
-					Hint: "Please run 'okteto context' to select one context"}
+					Hint: fmt.Sprintf("Please run 'okteto context' to select one context. Please run 'kubectl config use-context %s' if attempting to use a Kubernetes context that is a not an Okteto context", ctxOptions.Context),
+				}
```

Comment:
I would leave the previous message. You can select kubernetes context with `okteto ctx use`

Reply:
Done